m2cgen/cli.py

```diff
@@ -32,6 +32,7 @@
     "ruby": (m2cgen.export_to_ruby, ["indent", "function_name"]),
     "f_sharp": (m2cgen.export_to_f_sharp, ["indent", "function_name"]),
     "rust": (m2cgen.export_to_rust, ["indent", "function_name"]),
+    "elixir": (m2cgen.export_to_elixir, ["module_name", "indent", "function_name"]),
```

Comment:
Strictly speaking, no, module name is not required for a minimal standalone Elixir program.
But, practically speaking, it is needed.

Code that's not inside a module would be evaluated at compile time if compiling the code (and we need to compile it, otherwise it's too slow to run)

Also, if you're going to use the code in a real app, it needs to be in a module to be "imported" into other places (the module name will become the .beam filename, which is how the VM knows to find it at runtime)

Reply:
Oh, OK, thanks a lot for the explanation! I agree that module is a kind of required here to make generated code useful for users.